### ISSUE TITLE
Unable to copy / share links

### ISSUE THREAD

**Issue by peterbata** [CaramelFur/Picsur#56]
Hello. First time post. Picsur! is absolutely brilliant. The only issue that I am having is that I am unable to copy any of the generated links once I have uploaded the image. Error: Copying to clipboard failed. And, Sharing is not supported on your device (PC running Windows 10).  Would this have anything to do with the fact that I am accessing the app over my local nertwork using http and not https? Your time and assistance will be greatly appreciated. Thank you

**Comment by CaramelFur** [comment]
Yep, unfortunately the clipboard functionality is only available for "Secure Websites". So this is either https, or a localhost site. If you want to use http, you'll have to stick to manually selecting and copying.

**Comment by peterbata** [comment]
> Yep, unfortunately the clipboard functionality is only available for "Secure Websites". So this is either https, or a localhost site. If you want to use http, you'll have to stick to manually selecting and copying.

Thank you @CaramelFur Your response is greatly appreciated
